### ISSUE TITLE
Console. log

### ISSUE THREAD

**Issue by lavrenkov-sketch** [ccoreilly/vosk-browser#10]
![image](https://user-images.githubusercontent.com/52679491/130603883-1216ed10-5061-4513-9a4f-426aa33e6e91.png)
Hello, how disable this console.log ?

**Comment by ccoreilly** [comment]
It is currently not possible, but I'll implement it and publish a new version together with the changes in PR #11

**Comment by AnSrwn** [comment]
Is there any update to this issue? Can we already disable the console.log?

**Comment by ccoreilly** [comment]
@AnSrwn I published a new version (`0.0.6`) in which it is possible to set the log level. The default log level is 0 (info) and can be set either on model creation or during runtime:

```javascript
const model = new Model('model.tar.gz', -1);
// or
const model = await createModel('model.tar.gz', 3);
// or/and
model.setLogLevel(2);
```

Levels match those in vosk/kaldi:
```
-2: Error
-1: Warning
 0: Info
 1: Verbose
 2: More verbose
 3: Debug
```
